packages/sdk/src/chainConfig/plugin.ts

```diff
@@ -91,14 +89,12 @@ const chainPluginConfig: ChainPlugins = {
         strategyAddress: "0x574f934075D9b9392A8B0d1e0a4ADAD79B4cd16b",
         flywheels: [
           {
-            // FuseFlywheelCore_EPX.json
-            address: "0xC6431455AeE17a08D6409BdFB18c4bc73a4069E4",
+            rewardsCycleLength: 1, // TODO is this correct?
             // EPX rewards
             rewardToken: "0xaf41054c1487b0e5e2b9250c0332ecbce6ce9d71",
           },
           {
-            // FuseFlywheelCore_DDD.json
-            address: "0x851Cc0037B6923e60dC81Fa79Ac0799cC983492c",
+            rewardsCycleLength: 1, // TODO is this correct?
```

Comment:
@RedVeil was the one who initially set these, can't recall what the heuristics were